src-chrome-mv3/options.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Popup</title>
+    <style>
+      dt { white-space: nowrap }
+    </style>
+    <script src="options.js"></script>
+  </head>
+  <body>
+    <h1>Imageshare Search</h1>
+    <input type="search" id="search" name="imgs"
+    aria-label="Search for resource matches in Imageshare">
+    <label for="search">Search</label>
+    <button id="standard-search">Run Standard Search</button>
+    <fieldset>
```

Comment:
I'm not sure I'd use a `<dl>` to list a set of inputs; it might be more confusing than elaborating semantically. I'd suggest using a `<ul>` to group label/input groupings inside `<li>`; that way the AT user still gets a semantic count of how many inputs to expect.

Consider separating "Standard Search" and "Advanced Search" by respective headings, thus the heading layout becomes:

```
1. Imageshare search
  2. Standard search
  2. Advanced search
```

which then contextually groups the fields+button sets.

Reply:
sending you details in slack.

Reply:
Part 1 changes enacted in alerts-cross-browser
Part 2 code changes remove the need for this one